src/RageUtil.cpp

```diff
@@ -226,27 +236,27 @@ float HHMMSSToSeconds( const RString &sHHMMSS )
 	return fSeconds;
 }
 
-RString SecondsToHHMMSS( float fSecs )
+RString SecondsToHHMMSS(float fSecs)
 {
-	const int iMinsDisplay = (int)fSecs/60;
-	const int iSecsDisplay = (int)fSecs - iMinsDisplay*60;
-	RString sReturn = ssprintf( "%02d:%02d:%02d", iMinsDisplay/60, iMinsDisplay%60, iSecsDisplay );
+	const int iMinsDisplay = (int)fSecs / 60;

```

Comment:
Let's replace the c-style casts with `static_cast<int>` (here and below).

Reply:
Took care of these in a separate commit: https://github.com/itgmania/itgmania/pull/230/commits/ae1c515a5620c7f73c0373472f3068b8789391e8